File: Java/프로그래머스/1/12969. 직사각형 별찍기/직사각형 별찍기.java
import java.util.Scanner;
import java.util.stream.IntStream;
class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        StringBuilder sb = new StringBuilder();
        IntStream.range(0, a).forEach(e -> sb.append("*"));
        IntStream.range(0, b).forEach(e -> System.out.println(sb.toString()));
       
    }
}

File: Kotlin/프로그래머스/1/12969. 직사각형 별찍기/직사각형 별찍기.kt
fun main(args: Array<String>) {
    val (a, b) = readLine()!!.split(' ').map(String::toInt)
    var answer = StringBuilder()
    
    (1..b).forEach{
        (1..a).forEach{
            answer.append("*")
        }
        answer.append("\n")
    }
    print(answer)
}